tests/UI/Container_spec.js

```diff
@@ -95,7 +95,22 @@ describe("Container", function () {
         await capture.page(page, 'edit_url');
     });
 
+    it('should set isTagFireLimitAllowedInPreviewMode as active when set and save', async function(){
+        await page.evaluate(() => $('#isTagFireLimitAllowedInPreviewMode').click());
+        await page.waitForTimeout(250);
+        await page.waitForNetworkIdle();
+        await capture.page(page, 'edit_url_with_isTagFireLimitAllowedInPreviewMode');
+    });
+
+    it('should set isTagFireLimitAllowedInPreviewMode as active when set and save', async function(){
```

Comment:
Should this say `inactive` instead of `active`?

Reply:
yes :+1: